examples/marvel/lib/src/fake_marvel.dart

```diff
@@ -27,30 +27,30 @@ class FakeDio implements Dio {
   @override
   Future<Response<T>> get<T>(
     String path, {
-    required Map<String, Object?> queryParameters,
+    Map<String, dynamic>? queryParameters,
```

Comment:
Why change this?

Reply:
its just change from new **dio** version 4.0.0 
https://pub.dev/documentation/dio/latest/dio/Dio/get.html